src/cljc/rems/text.cljc

```diff
@@ -49,8 +50,13 @@
         (first (vals m)))))
```

Comment:
This `:default` can be removed.

Reply:
I think not yet, because licenses also have unlocalized titles that are used as :default keys. I will add a TODO comment here to remove this once all unlocalized titles are removed.

Reply:
👌